docs/setup_instructions/setup_overview.md

```diff
@@ -3,87 +3,42 @@ title: Pre-workshop setup
-
-For most of the installations below (all except Visual Studio Code), you will install software directly into the WSL 2 side of your computer, _not_ the Windows side.
-Even if you already have any of these softwares installed on the Windows side of your computer, you still need to install them on the WSL 2 side as described below.
-
-
-1. Install [Windows Subsystem for Linux 2](https://openscpca.readthedocs.io/en/latest/technical-setup/install-wsl-2/)
-    - Follow all instructions in the link above to install WSL 2, create your Ubuntu account, and update `apt`
-2. [Install GitKraken](https://openscpca.readthedocs.io/en/latest/technical-setup/install-a-git-client/#install-gitkraken-on-windows-with-wsl-2) into the WSL 2 side of your computer
-    - Follow the instructions in the link above to also sign into GitKraken with your GitHub account
-3. [Install Miniconda](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/setup-conda/#install-conda), a lightweight distribution of the package management software conda, into the WSL 2 side of your computer
-    - You do _not_ need to also set up conda; we will do this together during the workshop
-4. [Install R and RStudio Server](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/install-r-rstudio/) into the WSL 2 side of your computer
-5. Install some R packages we will use in the workshop by running the following command in R Console:
-    ```r
-    install.packages(c("renv", "tidyverse", "rmarkdown", "here", "optparse"))
-    ```
-6. Install [Visual Studio Code (VS Code)](https://code.visualstudio.com/), a popular text editor, into the Windows side of your computer
-    - [Following these instructions](https://code.visualstudio.com/docs/remote/wsl#_getting-started), you should also install and turn on the [WSL 2 extension](https://code.visualstudio.com/docs/remote/wsl#_getting-started) to be able to use VS Code from WSL 2
+We will _not_ have time to do all of the setup on the day of the workshop and accomplish all of our goals for the workshop!
+Please be aware this setup process may take 1-2 hours to complete if you are using Windows, and somewhat less on a Mac.
 
 
+{% if site.include_openrrp %}
+**Table of contents**
 
+* TOC goes here
+{:toc}
 
+### Only staying for "Reproducible Research Practices"?
 
-<!--
+If you are only staying for the first half of the workshop and _not_ for the portion about OpenScPCA, you can follow these setup instructions.
+_Otherwise, please proceed to the next section and follow all other instructions._
+{% endif %}
 
-Regular RRP installation instructions are commented out below.
+You will need to install R, RStudio, several R packages, GitKraken, and a suitable text editor, as well as any system dependencies required for these programs.
```

Comment:
I agree about only including TOC for the OpenRRP setup, but it probably makes sense to have a header here

```suggestion
## Workshop software requirements

You will need to install R, RStudio, several R packages, GitKraken, and a suitable text editor, as well as any system dependencies required for these programs.
```